docs/data-sources/enterprise_project.md

```diff
@@ -19,33 +19,68 @@ data "huaweicloud_enterprise_project" "test" {
 <!-- markdownlint-disable MD033 -->
 Service Name | Resource Name | Sub Resource Name
 ---- | --- | ---
+AOM | huaweicloud_aom_cmdb_application<br>huaweicloud_aom_cmdb_environment |
+APIG | huaweicloud_apig_instance |
 AS  | huaweicloud_as_group |
 BCS | huaweicloud_bcs_instance |
 BMS | huaweicloud_bms_instance |
 CBR | huaweicloud_cbr_vault |
+CC  | huaweicloud_cc_bandwidth_package<br>huaweicloud_cc_connection |
 CCE | huaweicloud_cce_cluster | huaweicloud_cce_node<br>huaweicloud_cce_node_pool<br>huaweicloud_cce_addon
+CCI | huaweicloud_cci_namespace |
+CCM | huaweicloud_ccm_private_certificate |
 CDM | huaweicloud_cdm_cluster |
 CDN | huaweicloud_cdn_domain |
-CES | huaweicloud_ces_alarmrule |
+CES | huaweicloud_ces_alarmrule<br>huaweicloud_ces_resource_group |
+CMDB | huaweicloud_aom_application<br>huaweicloud_aom_environment<br>huaweicloud_aom_resource_relationships |
+CodeArts | huaweicloud_codearts_project |
+CPH | huaweicloud_cph_server |
+CSE | huaweicloud_cse__microservice_engine |
+CSS | huaweicloud_css_cluster |
+DataArts | huaweicloud_dataarts_studio_instance |
+DBSS | huaweicloud_dbss_instance |
+DC  | huaweicloud_dc_virtual_gateway<br>huaweicloud_dc_virtual_interface |
 DCS | huaweicloud_dcs_instance |
-DDS | huaweicloud_dds_instance |
-DMS | huaweicloud_dms_kafka_instance<br>huaweicloud_dms_rabbitmq_instance |
+DDM | huaweicloud_ddm_instance |
+DDS | huaweicloud_dds_instance_v3 |
+Deprecated | huaweicloud_ecs_instance_v1 |
```

Comment:
the deprecated resource and the resource with **_v**[1-9] suffix should not be listed

Reply:
have changed